### ISSUE TITLE
REST API specification is not written.

### ISSUE THREAD

**Issue by HiroyukiYamashima** [personium/app-personium-calendar#47]
REST API specification write in Service JavaScript File.
Ex. updateVEvent.js

**Comment by shimono** [comment]
I agree that Engine service endpoint spec documents should be written somewhere and JavaScript file comment would be reasonable place.
